doc/examples/joystick/lib/static-joystick.dart

```diff
@@ -0,0 +1,67 @@
+import 'dart:math';
+import 'dart:ui';
+
+import 'package:flame/game.dart';
+import 'package:flame/gestures.dart';
+import 'package:flame/joystick.dart';
+import 'package:flame/sprite.dart';
+import 'package:flutter/cupertino.dart';
+
+import 'player.dart';
+
+class StaticJoystick extends Joystick {
```

Comment:
The filename should have an underscore, not a dash.

Reply:
Changed it, sorry.